File: Gestion_Angular_Front_end/src/app/classes/personel-dto.ts
export class PersonelDto {
  formatDate(arg0: Date): Date {
    throw new Error("Method not implemented.");
  }
  id: number= 0;
    nom: string= "";
    prenom: string= "";
    dateDeNaissance: string='';
    sexe: string= "";
    lieuDeNaissance: string= "";
    namePhoto: any;
    urlPhoto: any;
    username: any;
    numero: number= 0;
    email: string= "";
    password: string= "";
    roles: String[]= [];
}

File: Gestion_Spring_Back_end/src/main/java/com/odji/spring_back_end/dto/PersonelDto.java
package com.odji.spring_back_end.dto;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.odji.spring_back_end.model.Affectation;
import lombok.*;

import java.util.List;

@Data
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class PersonelDto {

    private Integer id;

    private String nom;

    private String prenom;

    private String dateDeNaissance;

    private String lieuDeNaissance;

    private String sexe;

    private String numero;


    private String email;

    private String password;

    private OptionDto role;

    @JsonIgnore
    private List<Affectation> affectation;
}
